src/static/css/index.css

```diff
@@ -17,6 +17,19 @@ header.alt-bg {
   background-color: transparent;
 }
 
+header .container {
+  max-width: none;
+  padding-left: 60px;
+  padding-right: 60px;
+}
+
+@media (max-width: 600px) {
+  header .container {
+    padding-left: 30px;
+    padding-right: 30px;
```

Comment:
Why 30px instead of 20px? It makes the menu appear weird on mobile for index page only:

![image](https://user-images.githubusercontent.com/10931297/69899923-e4880c00-1364-11ea-9dae-a1206ca4a849.png)

Reply:
That is correct, thanks for the heads up. 

`20` is the value I was aiming at, see the existing comment for the responsive nav: `/* overcome the parent container's 20px side padding */  `. I guess I got pretty tired at the end. Fixed it.